src/videojs-hls.js

```diff
@@ -950,6 +954,7 @@ videojs.Hls.prototype.drainBuffer = function(event) {
   // the playlist
   if (this.duration() !== Infinity && mediaIndex + 1 === playlist.segments.length) {
     this.mediaSource.endOfStream();
+    this.lastSeekedTime_ = undefined;
```

Comment:
Would we then not unset to null after a seek completes? Because the question I'm asking is, "has there been a seek since the video ended", and if video ending and seek completing do the same thing, I can't get the answer.